README.md

```diff
@@ -120,16 +135,9 @@ Once installed, `cd` into this repository's root directory and proceed to instal
 npm install
 ```
 
-### Quick Start
+##### Quick Start
 
-Once you have installed all necessary dependencies, create a `.env` file and replace the placeholder values with your credentials. Refer to the "Environment Variables" section below for further information.
-
-```bash
-cp sample.env .env
-# Open and edit .env in your text editor.
-```
-
-Finally, open three terminals and run the following commands:
+Open three terminals and run the following commands:
 
 ```bash
 # Terminal 1
```

Comment:
I believe we need a note  telling the user that they will need to stop the dm_app container with the command `docker stop dm_app`.  Otherwise `npm run front-end:watch` won't work as expected.